### ISSUE TITLE
[Feature] Multiple-select field values (breast type, ethnicity, nationality, etc)

### ISSUE THREAD

**Issue by echo6ix** [stashapp/stash-box#234]
**Is your feature request related to a problem? Please describe.**
A few fields would be more accurately represented with the capacity to have multiple values. This is related to #210 and expands upon it.

**Describe the solution you'd like**
Add multi-value support for:

1. Ethnicity
Applicable performer's could have combos like `Asian` and `Black` to accurately reflect them, instead of falling into one of the other or into `Mixed`, which seems like a catch-all for unknown mixed ethnicity.
2. Breast type
Many performers start their career with natural and change to augmented. How can we better reflect that?
a. Allowing multiple values
b. Allowing multiple values with an adjacent form for date ranges (i.e. "Natural, 2010 - 2015; Augmented: 2015 - 2019")
3. Nationality
Somewhat of an edge case, but there are performer's that have dual nationalities. This would apply to people who are born in country a, but raised and are a citizen of country b. This is really pedantic but I thought I'd mention it for accuracy's sake.
4. Hair color
See #210

**Comment by echo6ix** [comment]
Something like this...

![Screenshot 2022-07-30 134936](https://user-images.githubusercontent.com/37937507/181935566-6b4cc29a-cec0-4dd8-9fa7-413aed757b28.png)

Dates for breast type would obviously be optional, but allowing for `Natural` and `Augmented` would be more accurate than having to choose one or another when they've had both attributes. Same for hair color.

**Comment by sleetx** [comment]
This is a great idea and would be a huge improvement in how to track this information. I assume there will need to be some changes on the Stashapp end of things to fully utilize this.

**Comment by S3L3CT3DLoves** [comment]
I've been thinking about the best way to implement this. It will mean changin the DB to enable it, but that is not a problem.
What I'm wondering is, should we move the list of values to the DB at the same time?

In my mind there are two possibilities, we can make each field (Nationality, Ethnicity, Hair colour):
1. a list of freetext values, then define standard values in the code (like it currently is)
2. a pointer to a pre-defined value in the DB, and store the standard values in the DB

From my perspective, I would argue that the main value of Option 2 is the ability of each admin to update/select the values used in their StashBox. But I'm not sure that is actually very useful / necessary?



_Side note: The only change is Stash will be for the Country/Nationality, which is a dropdown in Stash. Everything else is freetext, so we can simply return a concat of the values._

**Comment by RollainKraus** [comment]
Spitballing some ideas here. I think some fields would benefit by having support for images. Breast Type being the obvious example here. But fields like hair color could have potential as well with having multiple images of the different hair colors/styles performers have had over their career. The only problem is the UI would likely get really messy if fields also supported images or new fields were added just for this

**Comment by discourse-stashapp** [comment]
This issue has been mentioned on **Stash**. There might be relevant details there:

https://discourse.stashapp.cc/t/multiple-select-performer-fields/4982/1

**Comment by echo6ix** [comment]
Recently I went back and updated performers I had initially synced from IAFD about five years ago. About 30% of them had new `Breast Type` and `Measurement` attributes.

TLDR these attributes are related and should be linked in some way going forward if multiple values are applied. Basically:

```json
{
  "malleable body attributes": [
    {
      "tag": "2015-2019",
      "breast type": "natural",
      "measurements": "aab-cc-dd"
    },
    {
      "tag": "2020-",
      "breast type": "augmented",
      "measurements": "aab-cc-dd"
    }
  ]
}
```
